src/gui/properties/propertieswidget.cpp

```diff
@@ -155,10 +155,10 @@ PropertiesWidget::PropertiesWidget(QWidget *parent, MainWindow *mainWindow, Tran
     m_tabBar = new PropTabBar();
     m_tabBar->setContentsMargins(0, 5, 0, 0);
     m_ui->verticalLayout->addLayout(m_tabBar);
-    connect(m_tabBar, SIGNAL(tabChanged(int)), m_ui->stackedProperties, SLOT(setCurrentIndex(int)));
-    connect(m_tabBar, SIGNAL(tabChanged(int)), this, SLOT(saveSettings()));
+    connect(m_tabBar, &PropTabBar::tabChanged, m_ui->stackedProperties, &QStackedWidget::setCurrentIndex);
+    connect(m_tabBar, &PropTabBar::tabChanged, this, &PropertiesWidget::saveSettings);
     connect(m_tabBar, SIGNAL(visibilityToggled(bool)), SLOT(setVisibility(bool)));
```

Comment:
you missed this one?
`connect(m_tabBar, &PropTabBar::visibilityToggled, this, &PropertiesWidget::setVisibility);`